mlrun/db/httpdb.py

```diff
@@ -2770,6 +2770,66 @@ def trigger_migrations(self) -> Optional[schemas.BackgroundTask]:
             )
         return None
 
+    def get_projects_dataschemas(self):
+        """Provide data schemas (feature store schemas) of MLRun projects as collection.
+
+        Examples::
+
+            ...
+            # connection to solution
+            db = mlrun.get_run_db()
+
+            # print data schema of all projects in JSON format
+            print(json.dumps(db.get_projects_dataschemas(),
+                ensure_ascii = False,
+                indent=2,
+                allow_nan=True)
+            )
+
+        :return:    Collection of data schemas (feature store schemas)
+        """
+
+        dataschema = []
+
+        # iteration via projects
+        for project in self.list_projects():
+            project_data_schema = {}
+            project_data_schema["project_name"] = project.name
+            if project.spec.description:
+                project_data_schema["project_description"] = project.spec.description
```

Comment:
for consistency between project schemas I think we should add `project_description` to each project even if its empty.
```suggestion
            project_data_schema["project_description"] = project.spec.description
```